tests/utils/compute_groundtruth.cpp

```diff
@@ -426,6 +459,11 @@ int main(int argc, char **argv) {
         "File name for the writing ground truth in binary format");
     desc.add_options()("K", po::value<uint64_t>(&K)->required(),
                        "Number of ground truth nearest neighbors to compute");
+    desc.add_options()("tags", po::value<bool>(&tags)->default_value(false),
+        "Whether to compute with tags. Default false.");
+    desc.add_options()("tags_file",
```

Comment:
Can we use just one additional flag "tags_file" ? If its an empty string as is the default, we dont use tags. If its non-empty, then we try to open the file and sanity check.